integration/querydsl/expressions/src/main/java/com/blazebit/persistence/querydsl/JPQLNextExpressions.java

```diff
@@ -32,6 +32,7 @@
 import com.querydsl.core.types.dsl.DateTimeExpression;
 import com.querydsl.core.types.dsl.Expressions;
 import com.querydsl.core.types.dsl.NumberExpression;
+import com.querydsl.core.types.dsl.SimpleOperation;
```

Comment:
[ERROR] src/main/java/com/blazebit/persistence/querydsl/JPQLNextExpressions.java:[24,8] (imports) UnusedImports: Unused import - com.querydsl.core.types.ExpressionUtils.
[ERROR] src/main/java/com/blazebit/persistence/querydsl/JPQLNextExpressions.java:[25,8] (imports) UnusedImports: Unused import - com.querydsl.core.types.Operation.

Reply:
Oops.

Reply:
Fixed. I actually reverted this change. In hindsight I think its more appropiate to put the alias within the bind operation rather than outside it. Besides `.as(alias)` , `SimpleOperation` also introduces methods like `isNull()`, `isNotNull()` which could lead to improper use (as the bind and the alias always need to be the outermost expression and are only allowed in some scenarios).